vendor/bat-native-ledger/src/bat/ledger/internal/media/vimeo.cc

```diff
@@ -99,7 +100,14 @@ std::string Vimeo::GetNameFromVideoPage(const std::string& data) {
     return "";
   }
 
-  return braveledger_media::ExtractData(data, ",\"display_name\":\"", "\"");
+  std::string publisher_name;
+  const std::string publisher_json_name =
+      braveledger_media::ExtractData(data, "\"display_name\":\"", "\"");
+  const std::string publisher_json = "{\"brave_publisher\":\"" +
+      publisher_json_name + "\"}";
+  braveledger_bat_helper::getJSONValue(
+      "brave_publisher", publisher_json, &publisher_name);
+  return publisher_name;
```

Comment:
Publisher pages have a `display_name` in the JSON blob that can be pulled from. Should be easier since the code point can be parsed. Otherwise, a way to `URIDecode` would have to be called out to RewardsService.